var/spack/repos/builtin/packages/py-hepstats/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyHepstats(PythonPackage):
+    """hepstats is a library for statistical inference aiming
+    to cover the needs in High Energy Physics.
+    It is part of the Scikit-HEP project.
+    """
+
+    homepage = "https://github.com/scikit-hep/hepstats"
+    pypi = "hepstats/hepstats-0.8.1.tar.gz"
+    maintainers("jonas-eschle")
+
+    license("BSD-3-Clause", checked_by="jonas-eschle")
+
+    tags = ["likelihood", "statistics", "inference", "fitting", "hep"]
+
+    version("0.8.1", sha256="ebb890496d7aebbf1d717de15d073be31d6775065308a4e0f263ed4051992b3f")
+
+    depends_on("python@3.9:3.12", type=("build", "run"), when="@0.8:")
```

Comment:
Curious. Where is the version cap coming from? (It's not in `pyproject.toml` or `setup.cfg`.)